pandas/_libs/src/inference.pyx

```diff
@@ -533,22 +533,6 @@ cpdef object infer_datetimelike_array(object arr):
     return 'mixed'
 
 
-cdef inline bint is_null_datetimelike(v):
```

Comment:
is there a reason these are not in nattype?  (and maybe namespaces to missing, but actual definition in nattype)

Reply:
I'd be fine putting this in nattype.  The one below I considered moving but decided against because it's only used once here in inference.pyx.